docs/asciidoctorj.adoc

```diff
@@ -1352,7 +1352,7 @@ public class UriIncludeProcessor extends IncludeProcessor { // <1>
 <3> `push_include` method inserts new content (retrieved from the url) in current position of document.
 
 [source]
-.Register an Inline macro processor
+.Register an Include processor
```

Comment:
Yes if you think thats correct. All other processors are written uppercase, however.